src/core/positioning/positioning.h

```diff
@@ -67,6 +67,9 @@ class Positioning : public QObject
 
     Q_PROPERTY( bool backgroundMode READ backgroundMode WRITE setBackgroundMode NOTIFY backgroundModeChanged )
 
+    Q_PROPERTY( double badAccuracyThreshold READ badAccuracyThreshold WRITE setBadAccuracyThreshold NOTIFY badAccuracyThresholdChanged )
+    Q_PROPERTY( double excellentAccuracyThreshold READ excellentAccuracyThreshold WRITE setExcellentAccuracyThreshold NOTIFY excellentAccuracyThresholdChanged )
```

Comment:
OK, the more I think about it, the more I'm realizing we do want to have a boolean property to decide whether averaged position should include or exclude bad readings.

Please add (above, alongside the two other average properties) a

Q_PROPERTY( bool averagePositionFilterAccuracy ... )

Then, when turned on will reject the AccuracyBad readings.

That way we'll be able to set the accuracy quality even when we don't want to enforce things, which will come in handy on the UI side of things.